site/cypress/e2e/Cypress-Feature/office_hours_queue.spec.js

```diff
@@ -304,7 +302,12 @@ describe('test office hours queue', () => {
         cy.get('@sound-switch').click();
         cy.get('@push-enabled').should('equal', false);
         cy.get('@audio-enabled').should('equal', false);
-
-        disableQueue();
+    });
+    it('Cleaning up', () => {
```

Comment:
Ideally making the state of the test should happen in a `before(()=>{});` block, as if the test is stopped mid-way through, it will fail if the cleanup is at the end, however if it is not possible to do it at the beginning it should at least be in an `after(()=>{});` block, and it should deal with the state if the test fails halfway through.